client/components/ReviewCard.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {Link} from 'react-router-dom'
+
+// MATERIAL UI IMPORTS
+import {withStyles} from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
```

Comment:
maybe we should just use the divider component from Material UI for reviews? Just a thought.